serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc

```diff
@@ -17,11 +19,36 @@ You can define a `dataInputSchema` as follows:
 }
 ----
 
-In the previous definition, the `schema` property is a URI, which holds the path to the JSON schema used to validate the workflow data input. The URI can be a classpath URI, a file, or an HTTP URL. If a classpath URI is specified, then the JSON schema file must be placed in the resources section of the project.
+In the previous definition, the `schema` property is a URI, which holds the path to the JSON schema used to validate the workflow data input. The URI can be a classpath URI, a file, or an HTTP URL. If a classpath URI is specified, then the JSON schema file must be placed in the resources section of the project or any other directory included in the classpath.
+
+`failOnValidationErrors` is an optional flag that indicates the behavior adopted when the input data does not match the specified JSON schema. If not specified or set to true, an exception will be thrown and flow execution will fail. If set to false, the flow will be executed and a log of level WARN with the validation errors will be printed.
+
+== Output schema
+
+Serverless Workflow specification does not support JSON output schema until version 0.9. Therefore {product_name} is implementing it as a link:{spec_doc_url}#extensions[Serverless Workflow specification extension]. Output schema is applied after workflow execution to verify that the output model has the expected format. It is also useful for Swagger generation purposes. 
+
+Similar to Input schema, you must specify the URL to the JSON schema, using `outputSchema` as follows:
+
+.`outputSchema` definition
+----
+"extensions" : [ {
+      "extensionid": "workflow-output-schema",
+      "outputSchema": { 
+         "schema" : "URL_to_json_schema",
+          "failOnValidationErrors": false
+     }
+  ]
+----
```

Comment:
```suggestion
[source,json,subs="attributes+"]
----
"extensions" : [ {
      "extensionid": "workflow-output-schema", <1>
      "outputSchema": { 
         "schema" : "URL_to_json_schema", <2>
          "failOnValidationErrors": false <3>
     }
  ]
----
```

Then you can describe each line below:

```asciidoc
1.
2.
3.
```

Reply:
hmm, good to know for future editions (I would certainly use it for executor guides), but in this case I prefer the old approach, since the content has already described in the previous section and I do not want to duplicate it

Reply:
I think Ricardo mainly meant to include the `[source,json,subs="attributes+"]` to have the highlighting.

Reply:
Ok added source hightlighting